executor/src/witgen/global_constraints.rs

```diff
@@ -249,14 +254,11 @@ fn propagate_constraints<T: FieldElement>(
                 false
             }
         }
-        // Bus receives are handled when they are matched with a send.
+        // Cannot derive a range constraint from a receive.
         Identity::BusReceive(..) => false,
         Identity::BusSend(send) => {
             let receive = send.try_match_static(bus_receives).unwrap();
```

Comment:
Why unwrap if it cannot match? Shouldn't we just ignore it in that case?

Reply:
Sure, but I don't really want to think about the dynamic case in this PR. There are many places were we unwrap now and have to handle it later. But I could change that we don't have `try_match_static` in the first place, only `match_static`, which panics. And then I can undo that in the next PR.